garak/probes/packagehallucination.py

```diff
@@ -109,3 +109,18 @@ class JavaScript(PackageHallucinationProbe):
     language_name = "JavaScript"
     goal = "generate JavaScript code importing non-existent npm packages"
     primary_detector = "packagehallucination.JavaScriptNpm"
+
+
+class Rust(PackageHallucinationProbe):
+    """Attempts to generate Rust code including hallucinated crates.io packages
+
+    Generators sometimes recommend importing non-existent crates.io packages. These
+    package names can be found by attackers and then squatted in public package
+    repositories, so that incorrect code from generators will start to run, silently
+    loading malicious squatted packages onto the machine. This is bad. This probe
+    checks whether a model will recommend code that uses non-existent crates.io packages."""
+
+    active = True
+    language_name = "Rust"
+    goal = "generate Rust code importing non-existent crates.io packages"
+    primary_detector = "packagehallucination.Rust"
```

Comment:
Incorrect name:
```
2024-09-04 12:04:26,330  WARNING  Exception instantiating garak.detectors.packagehallucination.Rust: module 'garak.detectors.packagehallucination' has no attribute 'Rust'
Traceback (most recent call last):
  File "/home/lderczynski/dev/garak/garak/_plugins.py", line 389, in load_plugin
    klass = getattr(mod, plugin_class_name)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'garak.detectors.packagehallucination' has no attribute 'Rust'
```
consider either renaming the detector, or doing this:
```suggestion
    primary_detector = "packagehallucination.RustCrates"
```

Reply:
Thanks for catching this @leondz ! Yeah, I wonder why the tests still passed?